### ISSUE TITLE
i18n support?

### ISSUE THREAD

**Issue by kiridaruma** [TopQuadrant/shacl#84]
SHACL validate result has `sh:resultMessage`. but its message only English now.

I think to need be able to pluggable for support i18n. but message is hardcoded now.

https://github.com/TopQuadrant/shacl/blob/master/src/main/java/org/topbraid/shacl/validation/java/AbstractClusiveConstraintExecutor.java

**Comment by HolgerKnublauch** [comment]
Yes I agree this needs to be addressed. I will keep the ticket open. Thanks.

**Comment by ashleycaselli** [comment]
@kiridaruma I created and linked a branch to this issue. Please, do not hesitate to propose possible solutions if you have something in mind already.

Let's see how @HolgerKnublauch sees this. In my opinion, the solution should be a pluggable one. In other words, I would like to avoid including  "all the possible" language files in the library and rather provide a way for plug&play the desired external language file/s.

**Comment by kiridaruma** [comment]
I'm sorry. I'd forgot this issue.
I don't have any code now. however I aggree to @ashleycaselli comment that "the solution should be a pluggable one."

I think it needs some argument that multiple language comment include to this repository or not.
My opinion is 
- include some language comment data to this repository.
- but provide pluggable interface from outside.

any question or comment?

FYI, my 1st language is Japanese. I can write Japanese comment.
